validator/src/main/resources/expected-data-template/dotnet/ec2/asg/remote-service-trace.mustache

```diff
@@ -0,0 +1,85 @@
+[{
+  "name": "^{{serviceName}}$",
+  "http": {
+    "request": {
+      "url": "^{{endpoint}}/remote-service$",
+      "method": "^GET$"
+    },
+    "response": {
+      "status": "^200$"
```

Comment:
Same here, won't comment everywhere but pls remove everywhere in validations

Reply:
OK removed from ASG and Default, pending test